docs/guides/fractional-gates.ipynb

```diff
@@ -0,0 +1,354 @@
+      "text/plain": [
+       "<Figure size 608.839x117.056 with 1 Axes>"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Decomposition of an RX(θ) gate using the IBM Quantum QPU basis\n",
+    "service = QiskitRuntimeService()\n",
+    "backend = service.backend('ibm_torino')\n",
+    "optimization_level = 3\n",
+    "pm = generate_preset_pass_manager(optimization_level, backend=backend)\n",
+    "transpiled_circuit = pm.run(qc)\n",
+    "transpiled_circuit.draw('mpl', idle_wires=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "6b256201-0237-4f63-91ff-fde1bb884804",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAPEAAACuCAYAAADnE+srAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjkuMiwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8hTgPZAAAACXBIWXMAAA9hAAAPYQGoP6dpAAANS0lEQVR4nO3dfUxU957H8fcgCvJkxbEdeaiFIl2k6HihWLltU7lodGmbmqyt0WC3hT82G1Z3Y8S2buOa3Y3FNulGTVNN3fSPZlkS++RFb9xurJFtuFYqsmyhGHmoDjK9jkAFFB9g9g+UWxQUkBn4nfm8EtPOnHPoFzpvz5kzwxyb1+v1IiLGCproAUTkwShiEcMpYhHDKWIRwyliEcMpYhHDKWIRwyliEcMpYhHDKWIRwyliEcMpYhHDKWIRwyliEcMpYhHDKWIRwyliEcMpYhHDKWIRwyliEcMpYhHDKWIRwyliEcMpYhHDKWIRwyliEcMpYhHDKWIRwyliEcMpYhHDKWIRwyliEcMpYhHDKWIRwyliEcMpYhHDKWIRwyliEcMpYhHDKWIRwyliEcMpYhHDKWIRwyliEcMpYhHDKWIRwyliEcMpYhHDKWIRwyliEcMpYhHDKWIRw1k+Yo/HQ1FREUlJSYSGhhIfH8/GjRvp7u4mPz8fm83Gnj17JnpMy+u6Ad9dhGOt8Mc/wS/XJ3oi6wie6AF86fTp06xcuRK32014eDjz58/nwoUL7Nq1i4aGBtra2gBwOp0TPaplNXXCfzbCYRdc7f3z/SFBsDwW1iTCEzMmckLz2bxer3eih/AFj8fDokWLcLlcbNq0iW3bthEZGQnAzp072bJlC8HBwfT29tLR0UFUVNREj2w537TC1u/het/w60yxwTsL4YVH/TmZtVg24rVr11JSUkJhYSG7d+++a7nT6aS6upqEhAQaGxsnZEYrO3ERNvwRekfw6LIBxRmQHeOPyazHks+J6+rqKC0txW63s2PHjiHXSU9PB2DhwoWD7m9qauKll14iMjKSmTNnsn79ei5duuSXua2izwvF/zuygAG8QHEN3LjHHluGZ8mIS0pK6OvrY926dURERAy5zvTp0+GOiDs7O1m6dCkul4uSkhL27dtHeXk
```

Comment:
I'm not sure, is "Pauli twirling" the same as "gate twirling". Namely, does it exclude measurement twirling? I'm not sure which of the three we want:
```
1. Gate twirling
2. Pauli twirling
3. Pauli gate twirling
```

Reply:
Yeah they're the same and in our docs on [error mitigation](https://docs.quantum.ibm.com/guides/error-mitigation-and-suppression-techniques#pauli-twirling) we make a distinction between Pauli twirling and measurement twirling. The link in the current draft of the page points toward the section on Pauli twirling. I can also make it more explicit in the list that pauli twirling is not supported, but measurement twirling *is*.